spec/hot_topic_spec.cr

```diff
@@ -26,6 +26,20 @@ describe HotTopic do
 
     EchoApp::JSONResponse.from_json(client.get("/json").body).path.should eq "/json"
   end
+
+  it "establishes a websocket" do
+    client = HotTopic.new(WebSocketEchoApp.new)
+    websocket = client.establish_ws("/echo/websocket")
```

Comment:
Hmm, the goal of this shard is to act as a body double for `HTTP::Client`, but we don't use that for websockets. Seems like it might make sense for this to be another class like `HotTopic::WebSocket` instead?